### PR TITLE
7177 multi cond print 2

### PR DESCRIPTION
**Pull request by DevneyHamilton**

### CODE REVIEW

**Comment by DevneyHamilton** [comment]
screen shot 2017-12-05 at 9 58 24 am
screen shot 2017-12-05 at 10 01 16 am

**Comment by DevneyHamilton** [comment]
Above comment has screenshots of what this looks like.

File: app/views/questionings/_printable.html.erb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <tr class="question <%= "depth-#{qing.depth}" %><%= qing.condition ? ' conditional' : '' %>">
   <td class="name">
     <% if qing.condition %>
```
Comment (hooverlunch) [inline app/views/questionings/_printable.html.erb]: Can we change this to use the new condition API? There is a method `display_conditionally?` that is perfect for here.

File: app/views/questionings/_printable.html.erb [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-      <div class="condition"><%= t("question.answer_only_caps") %> <%= qing.condition.to_s %></div>
+      <div class="condition"><%= t("question.answer_only_caps") %>
+        <%= qing.decorate.concatenated_conditions %></div>
```
Comment (hooverlunch) [inline app/views/questionings/_printable.html.erb]: Typically you'd indent the child tags on new lines, so:
```
<div>
  <tag1/>
  <tag2/>
</div>
```

unless there is some reason that having whitespace messes things up, but I don't think that's so here.

File: spec/decorators/qing_decorator_spec.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+describe QuestioningDecorator do
+  describe "display_if" do
+
+    let(:form) { create(:form, question_types: %w(integer integer integer integer)) }
+    let(:qing) { form.c.last }
+    let(:decorated_qing) { qing.decorate }
+    # let (:condition1) { Condition.new(ref_qing: form.c[0], op: "gt", value: "1") }
+    # let (:condition2) { Condition.new(ref_qing: form.c[1], op: "gt", value: "2") }
+    # let (:condition3) { Condition.new(ref_qing: form.c[2], op: "gt", value: "3") }
```
Comment (hooverlunch) [inline spec/decorators/qing_decorator_spec.rb]: Remove these?

File: spec/decorators/condition_decorator_spec.rb [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+        context "including code" do
+          let(:include_code) { true }
+          it { is_expected.to eq "Question #1 #{sel_q.code} Species is equal to \"Tulip\"" }
+        end
+      end
+    end
+  end
+
+  describe "multiple_conditions_display" do
```
Comment (hooverlunch) [inline spec/decorators/condition_decorator_spec.rb]: Is this old? I don't see this method defined anywhere

File: spec/decorators/qing_decorator_spec.rb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        decorated_qing.update_attribute(:display_if, "all_met")
+
+        expect(decorated_qing).to receive(:decorated_conditions).and_return(decorated_conditions)
+      end
+
+      it "returns only condition" do
+        expect(decorated_qing.concatenated_conditions).to eq "foo"
+      end
+    end
+  end
```
Comment (hooverlunch) [inline spec/decorators/qing_decorator_spec.rb]: Great job on these!

**Pull request merged**